### ISSUE TITLE
CLI only has `export` command

### ISSUE THREAD

**Issue by chasleslr** [gouline/dbt-metabase#77]
The docs mention `models` and `exposures` commands, but the CLI appears to only have the `export` command as of version 0.7.1.

For example, running `dbt-metabase exposures` returns the following:
```
usage: dbt-metabase [-h] --dbt_path PATH --mb_host HOST --mb_user USER --mb_password PASS [--mb_https HTTPS] --database DB --schema SCHEMA [--sync ENABLE] [--sync_timeout SECS] [--includes [MODELS [MODELS ...]]]
                    [--excludes [MODELS [MODELS ...]]]
                    {export}
dbt-metabase: error: argument command: invalid choice: 'exposures' (choose from 'export')
```

Steps to reproduce:
1. `pip install pipenv` (creates a new env, ignore if using venv)
2. `pipenv install dbt-metabase`
3. `pipenv run dbt-metabase exposures`

Version currently installed:
```
dbt-metabase==0.7.1
  - PyYAML [required: Any, installed: 6.0]
  - requests [required: Any, installed: 2.26.0]
    - certifi [required: >=2017.4.17, installed: 2021.10.8]
    - charset-normalizer [required: ~=2.0.0, installed: 2.0.7]
    - idna [required: >=2.5,<4, installed: 3.3]
    - urllib3 [required: >=1.21.1,<1.27, installed: 1.26.7]
```

**Comment by chasleslr** [comment]
Ah well, the docs on master appear to be for `v8.0.0` which is not yet released. I understand these commands are simply not yet available.

**Comment by gouline** [comment]
Version 0.8.0 is released as a release candidate [v0.8.0-rc5](https://github.com/gouline/dbt-metabase/releases/tag/v0.8.0-rc5). I'm planning the final release for this week.
